lib/error.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var util = require('util');
+var ErrorCat = require('error-cat');
+var log = require('./logger');
+
+/**
+ * Custom ErrorCat error handler.
+ * @class
+ */
+function PonosError () {
+  ErrorCat.apply(this, arguments);
+}
+util.inherits(PonosError, ErrorCat);
+
+/**
+ * Logs errors via default ponos logger.
+ * @param {error} err PonosError to log.
+ * @see error-cat
+ */
+PonosError.prototype.log = function (err) {
```

Comment:
If we end up going down the use all the things route then we'll definitely not need this.